src/cmdstan/stansummary.cpp

```diff
@@ -102,13 +131,30 @@ int main(int argc, const char *argv[]) {
     if (chains.param_name(i).length() > max_name_length)
       max_name_length = chains.param_name(i).length();
 
-  // Prepare values
-  static const int n = 9;
+  // Specify percentiles
+  Eigen::VectorXd probs = parse_probs(percentiles_spec);
+
+  // Prepare header
+  int num_cols = probs.size() + 6;
+  std::vector<std::string> headers(num_cols);
+  headers.at(0) = "Mean";
+  headers.at(1) = "MCSE";
+  headers.at(2) = "StdDev";
+  std::vector<std::string> percentiles;
+  boost::algorithm::split(percentiles, percentiles_spec, boost::is_any_of(", "),
+                          boost::token_compress_on);
+  uint idx = 3;
```

Comment:
This probably wasn't meant to be uint right?

Reply:
right.  thanks.